sonar-helm-for-iac/helm-imports.go

```diff
@@ -210,7 +209,9 @@ var sprigFunctionsWhitelist = []string{
 
 var sprigFunctions = initSprigFunctions()
 
-var generatedNamesCount = 0
+var includedNames = make(map[string]int)
+
+const includedMaxRecursion = 1000
```

Comment:
Where did this number come from? Just a gut feeling?

Reply:
Same value, as in Helm